modules/ui/TrainingTab.py

```diff
@@ -455,10 +455,15 @@ def __create_loss_frame(self, master, row, supports_vb_loss: bool):
                          tooltip="Minimum SNR gamma. Can help the model learn details more accurately. 0 disables, 20 maximum, ~5 is the usual setting")
         components.entry(frame, 3, 1, self.ui_state, "min_snr_gamma")
 
+        # Dropout Percentage
+        components.label(frame, 4, 0, "Dropout Percent",
```

Comment:
Same here. If it's not a percentage, the UI should use a different name. "probability" is already used in other places.